Casks/t/tdr-prism.rb

```diff
@@ -0,0 +1,25 @@
+cask "tdr-prism" do
+  version "1.0.7"
+  sha256 "627447da3fc7c8775d0ee93729482fd03b4ca0073a3acac3fae473c1018e5849"
+
+  url "https://www.tokyodawn.net/labs/Prism/#{version}/TDR%20Prism.zip"
+  name "TDR Prism"
+  desc "Frequency analyzer"
+  homepage "https://www.tokyodawn.net/tdr-prism/"
+
+  livecheck do
+    url :homepage
+    regex(%r{latest\sversion:\s<strong>(\d+(?:\.\d+)+)</strong>}i)
```

Comment:
```suggestion
    regex(%r{href=.*?Prism/v?(\d+(?:\.\d+)+)/TDR(%20|\s)Prism\.zip}i)
```

Updating livecheck to use the download link, from: https://github.com/Homebrew/homebrew-cask/pull/198103#discussion_r1917343904